Test flake: TestMySQLDatastoreWithTablePrefix/TestStats

```
--- FAIL: TestMySQLDatastoreWithTablePrefix (31.87s)
    --- FAIL: TestMySQLDatastoreWithTablePrefix/TestStats (0.22s)
        stats.go:26: 
            	Error Trace:	stats.go:26
            	            				datastore.go:49
            	Error:      	"0" is not greater than "0"
            	Test:       	TestMySQLDatastoreWithTablePrefix/TestStats
            	Messages:   	must report some relationships
FAIL
```

occasionally mysql seems to not be able to report stats even after an analyze. I've seen this a couple of times now, most recently on [Details](https://github.com/authzed/spicedb/runs/6508698626?check_suite_focus=true)

--- a/internal/datastore/mysql/stats.go
+++ b/internal/datastore/mysql/stats.go
@@ -55,7 +55,6 @@ func (mds *Datastore) Statistics(ctx context.Context) (datastore.Stats, error) {
 		if err != nil {
 			return datastore.Stats{}, err
 		}
-		var count uint64
 		err = mds.db.QueryRowContext(ctx, query, args...).Scan(&count)
 		if err != nil {
 			return datastore.Stats{}, err

